package.json

```diff
@@ -137,7 +137,7 @@
     "typescript": "~4.3.5"
   },
   "peerDependencies": {
-    "@nestjs/common": "^7.0.0",
+    "@nestjs/common": "^7.0.0 || ^8.0.0",
```

Comment:
If you are updating everything to v8 then there is no need to keep `^7.0.0` here is there?

Reply:
I tested with both v7v and v8 all tests are green, so actually, there is no need to drop support of v7 nest

Reply:
Then why upgrade the dependencies themselves? Just change the `peerDependencies`

Reply:
updated